python/test/Test_GNDStk_GridStyle.py

```diff
@@ -0,0 +1,86 @@
+# standard imports
```

Comment:
Some of this actually feels like highly redundant, over-the-top testing. :-) Like, it's true that value == none, and then it's false that value != none. But over-the-top testing is preferable to insufficient testing!

Reply:
True, but the test is designed to verify the following:
- the enumerator class can be imported in python without error
- the == and != operators on the enumerator work as they should
- all possible values of the enumerator have been bound properly (if I had forgotten a specific value in the bindings, I would have gotten an import error or something similar when trying to use it)

It might indeed look silly, but you can't imagine how many errors I have found in my code by testing things that might seem obvious or over the top. I can generally ensure quality on code we write because of it.